pages/about.html

```diff
@@ -4,9 +4,42 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>About</title>
+  <link href="/styles/style.css" rel="stylesheet">
+  <script src="https://kit.fontawesome.com/3e8fd03e4e.js"
+  crossorigin="anonymous"></script>
 </head>
 <body>
-  
+<header>
+<h1>Michelle's Portfolio Website</h1>
+  <nav>
+    <ul class="top-bottom-links">
+      <li>About</li>
+      <li><a href="/pages/index.html">Main</a></li>
+      <li><a href="/pages/portfolio.html">Portfolio</a></li>
+    </ul>
+  </nav>
+  </header>
+<section>
+  <h2>About Me</h2>
+  <img id="about-picture" src="https://media-exp1.licdn.com/dms/image/C5603AQFbXn1Y48bPig/profile-displayphoto-shrink_200_200/0/1619752757580?e=1643846400&v=beta&t=eRCcDIoyW_3sVQuxEoWmpLDH2nsR7xDJvEBFziUgZxQ" alt="profile picture">
+  <article class ="about-me"> 
```

Comment:
[article](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article) is another tag (like section) that prefers to have a heading nested within it.

Running your html through a validator would catch this.